package.json

```diff
@@ -14,8 +14,7 @@
   "dependencies": {
     "css-vendor": "^0.3.1",
     "debug": "^2.1.3",
-    "lodash.throttle": "^3.0.3",
-    "object.assign": "^4.0.1"
```

Comment:
This will make the code better. Babel has its own implementation of `Object.assign` that is shared in the whole compiled code. Adding the module `object.assign` is going to add multiple copies of the code that does the polyfill of the `Object.assign`.

Again, is it a problem having improvements in a PR even if not relative to the PR title? 
If you look at the single commits in this PR they already properly separated this and the `this.props.onOuterAction(event)` above.